feat: improve paid actors docs for devs (#723)
apify/apify-docs: sources/platform/actors/running/store.md

**Can I cancel my Actor rental?**

> During your trial or any time after that, **you can always cancel the Actor rental** so that you are not charged when your current Actor rental expires. If you choose to do so, you can always turn it back on later.

Reviewer: ```suggestion
**You can cancel the Actor rental** during your trial or any time after that so that don't get charged when your current Actor rental expires. You can always turn it back on later if you want.
```
